OlinkAnalyze/R/read_npx_parquet.R

```diff
@@ -43,7 +43,7 @@ read_npx_parquet <- function (filename) {
 
   # Check that all required parquet metadata is in place
   olink_parquet_metadata <- c("DataFileType",
-                              "ProductType")
```

Comment:
Will there be some data that is "ProductType" that we will need to support (like early adopters) or will all data that is delivered have "Product"?